docs/src/main/asciidoc/lifecycle.adoc

```diff
@@ -237,6 +237,63 @@ then the mode will be `DEVELOPMENT`, if you are running a JUnit test it will be
 Your application can get the launch mode by injecting the `io.quarkus.runtime.LaunchMode` enum into a CDI bean,
```

Comment:
```suggestion
Your application can get the _launch mode_ by injecting the `io.quarkus.runtime.LaunchMode` enum into a CDI bean,
```

Reply:
I think that this is irrelevant to the PR.

Reply:
It's a VALE check.